sites/public/src/ListingView.tsx

```diff
@@ -235,6 +239,17 @@ export const ListingView = (props: ListingProps) => {
     </>
   )
 
+  const additionalInformationCard = (cardTitle: string, cardData: string) => {
```

Comment:
We can pull this DRY into core

Reply:
Created an issue in Bloom off of this comment. https://github.com/CityOfDetroit/bloom/issues/1026